includes/wikia/models/NavigationModel.class.php

```diff
@@ -219,24 +183,24 @@ private function getTreeMemcKey( /* args */ ) {
 	 * @param array $maxChildrenAtLevel
 	 * @param bool $forContent
 	 * @param bool $refreshCache pass true to refresh the cache which stores parsed navigation tree
+	 *
 	 * @return Mixed|null
 	 */
-	public function getTree( $type, $source, Array $maxChildrenAtLevel = [], $forContent = false, $refreshCache = false ) {
+	public function getTree( $type, $source, Array $maxChildrenAtLevel = [], $refreshCache = false ) {
 		$menuData = WikiaDataAccess::cache(
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$menuData". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1488)